lib/recurly/pricing/calculations.js

```diff
@@ -237,3 +237,16 @@ Calculations.prototype.planPrice = function () {
 function decimal (prop) {
   this[prop] = (Math.round(Math.max(this[prop], 0) * 100) / 100).toFixed(2);
```

Comment:
I'm thinking I should fix this as well. It's less prone to side effects of floating point math problems because it's using a round and not a ceiling, but it is flawed nonetheless